ctod.dd

```diff
@@ -266,8 +266,7 @@ for (i = 0; i &lt; ARRAY_LENGTH; i++)
 $(H4 The D Way)
 
 ----------------------------
-int array[17];
-array[] = value;
+int[17] array = value;
```

Comment:
I think the assignment should stay separate. The initialization way might be a bit misleading, as it only works with fixed-size arrays.